docs/data/material/components/use-media-query/UseWidth.tsx

```diff
@@ -20,6 +20,7 @@ function useWidth() {
   const keys: readonly Breakpoint[] = [...theme.breakpoints.keys].reverse();
   return (
     keys.reduce((output: BreakpointOrNull, key: Breakpoint) => {
+      // TODO: uncomment once we enable eslint-plugin-react-compiler // eslint-disable-next-line react-compiler/react-compiler -- useMediaQuery is called inside callback
```

Comment:
I'm not sure if it is possible to achieve the same functionality by calling `useMediaQuery` outside of the callback. Also, I felt the effort wasn't worth it, so I added // eslint-disable-next-line react-compiler/react-compiler.